spec/checkPullRequestJobSpec.js

```diff
@@ -478,25 +721,29 @@ describe('Pull Request Job Spec', () => {
     it('should ping server jobs admin', () => {
       expect(github.issues.createComment).toHaveBeenCalled();
       const author = payloadData.payload.pull_request.user.login;
-      const formText = 'server jobs form'.link(
-        'https://goo.gl/forms/XIj00RJ2h5L55XzU2');
+      const formText = (
+        'server jobs form'.link('https://goo.gl/forms/XIj00RJ2h5L55XzU2'));
       const newLineFeed = '<br>';
       const wikiLinkText = (
         'this guide'.link(
           'https://github.com/oppia/oppia/wiki/Running-jobs-in-production' +
           '#submitting-a-pr-with-a-new-job'));
-      const jobRegistryLink = 'job registry'.link(
-        'https://github.com/oppia/oppia/blob/develop/core/jobs_registry.py');
-      const jobNameLink = 'OppiabotContributionsOneOffJob'.link(modifiedExistingJobFileObj.blob_url);
+      const jobRegistryLink = (
+        'job registry'.link(
+          'https://github.com/oppia/oppia/blob/develop/core/jobs_registry.py'));
+      const jobNameLink = 'OppiabotContributionsOneOffJob'.
+        link(modifiedExistingJobFileObj.blob_url);
```

Comment:
We should either do what was done on line 731 or change this to 
```
const jobNameLink = 'OppiabotContributionsOneOffJob'.link(
  modifiedExistingJobFileObj.blob_url
);
```

Reply:
@vojtechjelinek

Reply:
I don't think so. I'm saying we should choose one style, and use that in all places.

Reply:
Done